src/RoslynInsertionTool/RoslynInsertionTool.Commandline/RIT.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <RootNamespace>RoslynInsertionTool</RootNamespace>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
Pipelines may need to be manually updated after this change. Check the OneOffInsertion step in https://devdiv.visualstudio.com/DevDiv/_releaseDefinition?definitionId=2114&_a=definition-tasks&environmentId=11199

Ideally, we could also change packaging to also put the tool executables and scripts in some simpler path that won't change when we change the TFM.

Reply:
Downgraded to v4.9.4 so we can keep net46 tfm for now